shtdlib.sh

```diff
@@ -138,9 +141,12 @@ elif [ "${os_family}" == 'Alpine' ]; then
     os_name='alpine'
 fi
 
-# Store local IP addresses (not localhost)
+# Store local IP addresses (not localhost), if it is not virtualized in Docker
```

Comment:
Perhaps use more generic terms like container since we're checking for lxc and docker :)

Reply:
I removed the check for `Docker` virtualization platform. Instead it will just return an empty string if it can't find any, which I think is a more generic solution to a machine without a network.

Reply:
I think we still need to update that comment no?

Reply:
It's already been changed, I don't think it updates existing threads.